src/content/docs/discovery/architecture.mdx

```diff
@@ -0,0 +1,168 @@
+import Task from '@components/Task.astro';
+import Tasks from '@components/Tasks.astro';
+
+If you're looking for guidance about launching a production-ready, headless Adobe Commerce storefront with [Edge Delivery Services](https://www.aem.live/), you're in the right place!
+
+## Big picture
+
+Headless Adobe Commerce storefronts on Edge Delivery Services are built using a composable architecture with domain-driven commerce components called [dropins](/dropins/all/introduction/). This architecture allows you to build and deploy a storefront that is composed of multiple Adobe services, each with its own responsibility. Dropins are connected through APIs and can be developed, tested, and deployed independently for faster development cycles.
+
+Dropins rely on [Adobe Commerce](https://developer.adobe.com/commerce/webapi/graphql/) and [Catalog Service](https://developer.adobe.com/commerce/services/graphql/catalog-service/) APIs to provide data and functionality. These components are designed to be reusable and can be shared across multiple projects. They are integrated out-of-the-box with Edge Delivery Services through the [Commerce boilerplate](https://github.com/hlxsites/aem-boilerplate-commerce). Adobe Commerce storefronts on Edge Delivery Services are compatible with doc-based authoring, which allows business users to create and manage content without developer intervention.
+
+The following diagram illustrates the composable architecture of a headless Adobe Commerce storefront on Edge Delivery Services:
+
+<Diagram caption="Composable architecture of a headless Adobe Commerce storefront on Edge Delivery Services.">
+  ![Composable architecture of a headless Adobe Commerce storefront on Edge Delivery Services.](@images/implementation/headless-architecture-v4.svg)
+</Diagram>
+
+:::note
+Although not represented in this diagram, [API Mesh for Adobe Developer App Builder](https://developer.adobe.com/graphql-mesh-gateway/) is an option for integrating multiple APIs into a single API endpoint. It can be used to aggregate data from multiple APIs and provide a single endpoint for the storefront to consume.
+:::
+
+## Storefront
+
+The storefront is the front-end layer of your Adobe Commerce site. It is responsible for rendering the user interface and providing a seamless shopping experience for customers. The storefront is built using a combination of dropins, widgets, and front-end blocks that are connected to Adobe Commerce and other services through APIs and hosted on Edge Delivery Services.
+
+The Commerce boilerplate provides an integrated set of dropins and widgets that you can use to build a headless Adobe Commerce storefront on Edge Delivery Services. See [Create your storefront](/get-started/) for more information.
+
+:::note
+If you have an existing Adobe Commerce storefront (Luma or PWA Studio), you can migrate it to Edge Delivery Services in phases using a [Luma Bridge](/discovery/luma-bridge/).
```

Comment:
you have an option to reuse parts of the existing storefront with your Edge Delivery storefront by connecting them using [Luma Bridge](/discovery/luma-bridge/).